pkg/fqdn/namemanager/api.go

```diff
@@ -276,6 +276,31 @@ func (h *getFQDNNamesHandler) Handle(params GetFqdnNamesParams) middleware.Respo
 	return NewGetFqdnNamesOK().WithPayload(payload)
 }
 
+type getFQDNGCCacheHandler struct {
+	*manager
+}
+
+func (h *getFQDNGCCacheHandler) Handle(params GetFqdnGccacheParams) middleware.Responder {
+	if h.gcCache == nil {
+		return NewGetFqdnGccacheNotFound()
+	}
+
+	entries := h.gcCache.List()
+	if len(entries) == 0 {
+		return NewGetFqdnGccacheNotFound()
+	}
```

Comment:
Hm.. Since there is actually no query/filter parameter here (with the other handlers in thsi file do), I'm wondering if it makes sense to return 404 for the empty list here. I think returning `Ok` with an empty payload might actually be nicer.

Reply:
agreed! updated